app/models/manageiq/providers/openstack/cloud_manager/refresh_parser.rb

```diff
@@ -198,17 +197,16 @@ def parse_flavor(flavor)
       uid = flavor.id
 
       new_result = {
-        :type    => "ManageIQ::Providers::Openstack::CloudManager::Flavor",
-        :ems_ref => uid,
-        :name    => flavor.name,
-        :enabled => !flavor.disabled,
-        :cpus    => flavor.vcpus,
-        :memory  => flavor.ram.megabytes,
-        :disk_size      => flavor.disk.to_i.gigabytes,
-        :disk_count     => flavor.disk.to_i.gigabytes > 0 ? 1 : 0,
-        # Extra keys
-        :ephemeral_disk => flavor.ephemeral.to_i.gigabytes,
-        :swap_disk      => flavor.swap.to_i.megabytes
+        :type                 => "ManageIQ::Providers::Openstack::CloudManager::Flavor",
+        :ems_ref              => uid,
+        :name                 => flavor.name,
+        :enabled              => !flavor.disabled,
+        :cpus                 => flavor.vcpus,
+        :memory               => flavor.ram.megabytes,
+        :root_disk_size       => flavor.disk.to_i.gigabytes,
+        :swap_disk_size       => flavor.swap.to_i.megabytes,
+        :ephemeral_disk_size  => flavor.ephemeral.to_i.gigabytes,
```

Comment:
@Fryguy I remember where I saw 0 for ephemeral_disk_size. It's in our cassette. So we must have collected a 0 at one time.